velox/exec/OutputBuffer.h

```diff
@@ -74,6 +74,36 @@ class ArbitraryBuffer {
   std::deque<std::shared_ptr<SerializedPage>> pages_;
 };
 
+/// The stats of a DestinationBuffer. The data transferred by the buffer has
```

Comment:
There is a bit of inconsistency in naming throughout this change. Multiple different names are used for the same thing. Many comments contain repetitive / redundant text. Let's fix that. I'll point out a few examples and hopefully it will be clear what I mean. 

BufferStats -> DestinationBufferStats

> The stats of a DestinationBuffer.

This statement is redundant. It just repeats the struct name.

Reply:
done.